x/migration/module/cmd/collect_morse_accounts.go

```diff
@@ -270,9 +289,25 @@ func collectInputSupplierStakes(inputState *migrationtypes.MorseStateExport, mor
 	for exportSupplierIdx, exportSupplier := range inputState.AppState.Pos.Validators {
 		supplierAddr := exportSupplier.Address.String()
 
-		// DEV_NOTE: An account SHOULD exist for each actor.
 		if !morseWorkspace.hasAccount(supplierAddr) {
-			return ErrMorseExportState.Wrapf("account not found corresponding to supplier with address %q", supplierAddr)
+			logger.Logger.Warn().
+				Str("supplier_address", supplierAddr).
+				Msg("no account found for supplier")
+
+			// DEV_NOTE: If no auth account was found for this supplier, create a new one.
+			newSupplierAccount := &migrationtypes.MorseAuthAccount{
```

Comment:
Ditto comment above.

Reply:
This, I can confirm, definitely does happen in snapshot data. It seems like there are several Morse applications and suppliers which have no corresponding auth account; i.e. no unstaked balances. As of the current state of this branch, these accounts would be eligible to claim on Shannon as their respective actor type, but they will not have any unstaked balance. For practical purposes, this means that they will require intervention from an external account on Shannon in order to do anything (unstake, restake, etc.); however, this may also be the case for these accounts on Morse as well.

Reply:
So in some cases you can :
- Definitely confirm something does / does not happen
- Not sure if/why it does / does not happen

My ask is to add a TODO_MAINNET_MIGRATION in both cases with the detail and #PUC.